### PR TITLE
Exposed Ionic server timeout as a launch.json property

### PR DESCRIPTION
**Pull request by guillaumejenkins**
Addresses #134

### CODE REVIEW

**Comment by MSLaguana** [comment]
Looks good to me.

File: package.json [hunk 1]
```diff
@@ -287,6 +287,11 @@
                                 "type": "string",
                                 "description": "The IP address that Ionic should use for the live reload server"
                             },
+                            "devServerTimeout": {
+                                "type": "number",
+                                "description": "Timeout for starting the Ionic dev server when serving to the browser or running with Ionic live reload enabled.",
```
Comment (dlebu) [inline package.json]: We should mention the unit as well - 

> Timeout, in milliseconds, for starting...

File: src/debugger/cordovaDebugAdapter.ts [hunk 2]
```diff
@@ -596,7 +596,7 @@ export class CordovaDebugAdapter extends WebKitDebugAdapter {
         let errorRegex: RegExp = /error:.*/i;
         let serverReady: boolean = false;
         let appReady: boolean = false;
-        let serverReadyTimeout: number = 10000;
+        let serverReadyTimeout: number = launchArgs.devServerTimeout || 10000;
```
Comment (dlebu) [inline src/debugger/cordovaDebugAdapter.ts]: Would we want to bump up the default as well?

**Comment by AlGantori** [comment]
I just started using the cordova tools extension for ionic2,  and I see time out (20000 ms) where exactly in the launch.json I need to make the change.  Please indicate the syntax to bump this default up. 
Do I need to also install the "Chrome extension" ? Thank you.

**Comment by AlGantori** [comment]
To whoever stumbles here
OK, I browsed the "Files changed" above and guessed, I added
, "devServerTimeout" : 30000
and this time it worked and launched on Chrome to my amazement (Firefox is my default browser)
Do I need to also install the "Chrome extension" ? Thank you.

**Comment by MSLaguana** [comment]
You don't need the chrome extension installed unless you want to manually launch/attach chrome. Our extension uses the chrome debugging extension internally to support Cordova scenarios. We don't support debugging firefox since the debugging protocols are different.

**Pull request merged**